scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -239,6 +245,7 @@ const processRelatedAssetIds = async (
     assetData[assetId].relatedAssetKey = relatedAssetKey
   }
   relatedAssetIndex[relatedAssetKey] = relatedAssetIds
+  return
```

Comment:
Paranoia return out of promise to ensure resolution, shouldn't be needed but...

Reply:
this would be bigly news to me if this was needed for any reason lol :P

Reply:
Yeah 99% sure we don't need this but might as well!